policybot/dashboard/topics/maintainers/topic.go

```diff
@@ -190,35 +141,37 @@ func (t *topic) handleMaintainersListHTML(w http.ResponseWriter, r *http.Request
 		} else if filter&emeritus != 0 {
 			title = "Emeritus Maintainers"
 		}
-
-		t.context.RenderHTML(w, title, string(MustAsset("list.html")), "")
-	} else {
-		t.context.RenderHTML(w, "", string(MustAsset("list.html")), "")
 	}
+
+	return types.RenderInfo{
+		Title:   title,
+		Content: string(MustAsset("list.html")),
+	}, nil
 }
 
-func (t *topic) handleMaintainerListJSON(w http.ResponseWriter, r *http.Request) {
-	orgLogin := r.URL.Query().Get("org")
+func (t *Maintainers) ListAPI(w http.ResponseWriter, req *http.Request) {
```

Comment:
It looks like this is an API that implements List functionality, rather than for listing APIs. Would it make sense to call this `List` or `ListMaintainers`, or `HandleList`?